pkg/statistics/sample.go

```diff
@@ -61,15 +62,36 @@ func CopySampleItems(items []*SampleItem) []*SampleItem {
 	return n
 }
 
-func sortSampleItems(sc *stmtctx.StatementContext, items []*SampleItem) error {
+func sortDatumByBinary(items []types.Datum, getComparedBytes func(datum types.Datum) ([]byte, error)) error {
```

Comment:
could you comment on why binary format matters rather than simple item value comparison? for later reviewer's quick grabbing

Reply:
It is only for test. so i move it into the test file and add comment to it.